l10n/bundle.l10n.json

```diff
@@ -18,9 +18,10 @@
   "Show browser mode does not work in remote vscode": "Show browser mode does not work in remote vscode",
   "Show browser": "Show browser",
   "Show trace viewer": "Show trace viewer",
+  "embedded trace viewer": "embedded trace viewer",
```

Comment:
Unused, revert

Reply:
Addressed in b8a3d8b105bfd17b1c4c1dd8b4263723356ef219